.changes/next-release/enhancement-eksgettoken-91863.json

```diff
@@ -0,0 +1,5 @@
+{
+  "type": "enhancement",
+  "category": "eks get-token",
+  "description": "Add support to respect env var KUBERNETES_EXEC_INFO in eks get-token"
```

Comment:
For the description, let's update it to include API version somewhere in it like:

> Add support for respecting API version found in KUBERNETES_EXEC_INFO environment variable